readme.md

```diff
@@ -1398,18 +1450,21 @@ In 22.1, use the following CSS rule:
 }
 
```

Comment:
On line 1393 above, please change "22.1" to "v22.1":

`In v22.1, use the following CSS rule:`